cli/tsc/dts/lib.deno.window.d.ts

```diff
@@ -97,9 +97,65 @@ declare var onunload: ((this: Window, ev: Event) => any) | null;
 declare var onunhandledrejection:
   | ((this: Window, ev: PromiseRejectionEvent) => any)
   | null;
-/** @category Storage */
+/** 
+ * Deno's {@linkcode localStorage} API provides a way to store key-value pairs in a web-like environment, 
```

Comment:
i wouldnt link to itself, i'd find it a bit confusing if there is a link that goes to the same page
```suggestion
 * Deno's `localStorage` API provides a way to store key-value pairs in a web-like environment, 
```